script.js

```diff
@@ -1,4 +1,213 @@
+// Steps
+// 1) Generate Random Number of two dices max 6, return output for player 1
+// 2) Player choose which number to go first, return order output for player 1
+// 3) Generate Random Number of two dices max 6, return output for player 2
+// 4) Player choose which number to go first, return order output for player 2
+// 5) Compare two order outputs, return output which player wins
+// 6) Reset the Game without refreshing the browser page
+
+var gameMode = "startGame";
+var myOutputValue = "";
+var getDiceNumber1;
+var getDiceNumber2;
+var getDiceNumber3;
+var getDiceNumber4;
+var playerNumberCombination1;
+var playerNumberCombination2;
+
+//generate random dicenumber
+var generateRandomDiceNumber = function () {
+  var randomDecimal = Math.random() * 6;
+  var diceNumber = Math.ceil(randomDecimal);
+  return diceNumber;
+};
+
+//player 1 get two dice numbers
+var getTwoDiceRollResult1 = function () {
```

Comment:
You can try to use one single function for both player 1 and player 2 when they roll two dice

Reply:
Especially since if you look at `getTwoDiceRollResult1` and `getTwoDiceRollResult2`, they have quite alot of repeated code.